schemas/context/profile.schema.json

```diff
@@ -65,6 +65,11 @@
           "$ref": "https://ns.adobe.com/xdm/context/phonenumber",
           "description": "Mobile phone number.\n"
         },
+        "xdm:faxPhone": {
```

Comment:
This should not be a top level element in xdm - it should be inside something else, say Address or Phone.  Looks like there are a bunch of these in this particular schema.  @asteed can you resole?

Reply:
@lrosenthol Adding `faxPhone` is consistent with the other properties, so I would not reject it because of this.

But: I agree that it is an area that will lead to channel-creep (is `whatsAppPhone` next?). If you agree, let's file an issue to have channel-encoded contact coordinates and opt-in/out instead.

Reply:
@trieloff @lrosenthol : Can we have a resolution of the above xdm field from your side?

Reply:
@trieloff @lrosenthol @aarjain I have created https://github.com/adobe/xdm/issues/269 for this. We can take this discussion there and let this be unblocked for now. WDYT?

Reply:
Sounds good, @saurabhere.

@lrosenthol – do you agree that we can merge even with #269 unresolved (which is an XDM-wide concern)

Reply:
yes @trieloff I am fine since we have the bigger issue identified separately and we agree not to "creep" in any more